hebiros_advanced_examples/launch/mapping_navigation/param/base_local_planner_params.yaml

```diff
@@ -0,0 +1,43 @@
+TrajectoryPlannerROS:
+
+# Robot Configuration Parameters
+  max_vel_x: 2.0 #0.5
+  min_vel_x: -1.0
+  max_vel_y: 2.0
+  min_vel_y: -1.0
+
+ # y_vels: [-1.0,-1.0,1.0,1.0]
```

Comment:
In a future PR, it'd be nice to clean up a lot of these commented out lines, unless they serve a purpose, and document anything that's unclear.  Also, I'd probably change the min/max vel theta unless there is a reason to keep them 0.

One thing that really confused me -- why is max +2 and min -1 for the robot?  Should these not be the same magnitude?